Dockerfile

```diff
@@ -26,8 +30,13 @@ RUN apt-get update && apt-get install -y unzip --no-install-recommends && \
 RUN chmod +x /usr/bin/dep
 
 WORKDIR /go/src/github.com/darthhater/bored-board-service
+
 COPY . .
 
+RUN mkdir .keys && \
+    openssl genrsa -out .keys/app.rsa 1024 && \
+    openssl rsa -in .keys/app.rsa -pubout > .keys/app.rsa.pub
```

Comment:
This will need modified to:
`openssl rsa -in ${PRIVATE_KEY_PATH} -pubout > ${PUBLIC_KEY_PATH}`